libcloudforensics/providers/kubernetes/base.py

```diff
@@ -221,29 +163,28 @@ def ListPods(self, namespace: Optional[str] = None) -> List['K8sPod']:
 
     Returns:
       List[K8sPod]: The list of the node's pods for the namespace, or in all
-        namespaces if none is specified.
+          namespaces if none is specified.
     """
     api = self._Api(client.CoreV1Api)
 
     # The pods must be running, and must be on this node. The selectors here
     # are as per the API calls in `kubectl describe node NODE_NAME`.
     running_on_node_selector = selector.K8sSelector(
-      selector.K8sSelector.Node(self.name),
-      selector.K8sSelector.Running(),
+        selector.K8sSelector.Node(self.name),
+        selector.K8sSelector.Running(),
     )
 
     if namespace is not None:
```

Comment:
```suggestion
    if namespace:
```

Reply:
I want to make a difference between the cases when the method was called with the `namespace` keyword specified and when it is not specified.

Here if a user specifies a namespace with the keyword argument `namespace=''`, I believe that since a namespace was specified, the namespaced API call should be used. If we change to `if namespace:` this will not be the case, as `''` evaluates to `False`.